docs/application/native/get-started/iot/first-app.md

```diff
@@ -0,0 +1,292 @@
+
+    - **Features**: Define required software and hardware features. This information is used for application filtering in Tizen Store.
+
+    - **Privileges**: Define privileges for the security-sensitive APIs or API groups accessed and used by the application.
+
+    - **Localization**: Define localized values for the application label, description, and icon.
+
+    - **Advanced**: Define advanced features, such as application metadata, data control for services, application control functionalities, and account details.
+
+    - **Source**: View and edit the source code of the `tizen-manifest.xml` file. Changes made and saved on the other tabs are reflected in the source code and vice versa.
+
+        > [!NOTE]
+        > The `tizen-manifest.xml` file must conform to both the XML file format and the Tizen Native application specification requirements. Editing the file in the **Source** tab is intended for advanced users only.
+
+3. To save any changes, select **File > Save All** in the Tizen Studio menu.
+
+For more information on configuring the application, see [Setting the Application Manifest](../../tutorials/process/setting-properties.md#manifest).
+
+### Understanding the Source Code
+
+Pay attention to the life-cycle callbacks in the application source code to understand how the application works.
+
+The `main()` function in the `src/service.c` file is used to register callbacks that manage specific parts of the application life-cycle:
+
+-   `service_app_create`
+    -   Called when the application process starts.
+-   `service_app_terminate`
+    -   Called while the application process is terminating.
+    -   Called after the main loop quits.
+-   `service_app_control`
+    -   Called after the `service_app_create` callback when the process starts.
+    -   Called when a launch request is received while the process is running.
+    -   Can receive `service_app_control` data (parameters for launching the application).
+    -   Used to implement parameter-specific actions of the application.
+
+```
+int
+main(int argc, char *argv[])
+{
+    char ad[50] = {0,};
+    service_app_lifecycle_callback_s event_callback;
+    app_event_handler_h handlers[5] = {NULL, };
+
+    event_callback.create = service_app_create;
+    event_callback.terminate = service_app_terminate;
+    event_callback.app_control = service_app_control;
+
+    return service_app_main(argc, argv, &event_callback, ad);
+}
+```
+
+### Adding the log
+
+You can print logs using the Dlog API in the Tizen native application.
+For more information on **Dlog**, see [Dlog](../../../../iot/api/5.5/tizen-iot-headless/group__CAPI__SYSTEM__DLOG.html) API.
+
+In this example, to confirm that the IoT native service application has been launched successfully, add the **Hello Tizen** as a debug log.
+
+You can add the debug log as follows:
+
+1. Confirm that **#include<dlog.h>** is included, and **LOG_TAG** is defined in `inc/service.h`.
```

Comment:
Please fix the code alignment for step 1 and step 2

Reply:
Yes, I have been a little confused about indentation. Thank you for your clear explanation.